docs/content/installation/helm.md

```diff
@@ -90,13 +90,43 @@ PGO will check for updates upon startup and once every 24 hours. Any errors in c
 
 ## Upgrade and Uninstall
 
-And once PGO has been installed, it can then be upgraded and uninstalled using applicable `helm`
-commands:
+Once PGO has been installed, it can then be upgraded using the `helm upgrade` command.
+However, before running the `upgrade` command, any CustomResourceDefinitions (CRDs) must first be
+manually updated (this is specifically due to a [design decision in Helm v3][helm-crd-limits],
+in which any CRDs in the Helm chart are only applied when using the `helm install` command).
+
+[helm-crd-limits]: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/
+
+If you would like, before upgrading the CRDs, you can review the changes with
+`kubectl diff`. They can be verbose, so a pager like `less` may be useful:
+
+```shell
+kubectl diff -f helm/install/crds | less
+```
+
+Use the following command to update the CRDs using
+[server-side apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/)
+_before_ running `helm upgrade`. The `--force-conflicts` flag tells Kubernetes that you recognize
+Helm created the CRDs during `helm install`.
+
+```shell
+kubectl apply --server-side --force-conflicts -f helm/install/crds
+```
+
+Then, perform the upgrade using Helm:
 
 ```shell
 helm upgrade <name> -n <namespace> helm/install
 ```
 
+To uninstall PGO, remove all your PostgresCluster objects, then use the `helm uninstall` command:
```

Comment:
Is is necessary to remove all PostgresCluster objects?

In the past I've def seen requests to uninstall PGO, but leave all clusters in place.

Reply:
🤔 If the controller goes away first, nothing will handle the finalizers on those PostgresClusters.

So it's a question, I guess, of whether someone wants to give up completely on PGO or just undo something done by Helm?

Reply:
Agreed.  Looking back (I distinctly remember this coming up for the Ansible installer for v4 in the past), we've seen it both ways - i.e. some users looking for _everything_ to be installed, and others looking for just the PGO Deployment to go away (but not the PG clusters).

I'm good with assuming `uninstall` means _everything_ for now, especially considering the finalizers :+1: